src/iuse.cpp

```diff
@@ -568,18 +568,14 @@ int iuse::smoking( player *p, item *it, bool, const tripoint & )
     }
 
     item cig;
-    if( it->typeId() == itype_cig ) {
+    if( it->typeId() == itype_cig || it->typeId() == itype_handrolled_cig ) {
```

Comment:
That's looks like a great change, but after the codeblocks are merged the comment is no longer necessary in the code.

Reply:
True, but I figure that, at least for a while, having it there isn't a bad idea in case someone looks at it, remembers what it looked like before, and asks.